src/prostredi/preklad_programu.md

```diff
@@ -64,9 +64,10 @@ Program by měl na výstup vytisknout text `Hello world!`.
     Pokud překlad proběhne úspěšně, tak operátor `&&` zajistí spuštění následujícího příkazu.
 - Pokud nechcete příkazy v terminálu psát neustále dokola, šipkou nahoru (&#8593;) můžete vyvolat nedávno
 spuštěné příkazy v terminálu.
+- Můžete používat i terminál vestavený přímo ve `Visual Studio Code` (`View -> Terminal`).
```

Comment:
Build in terminal often suppresses Address Sanitizer. In current "demo" project, when app is run, it opens a separate terminal to ensure everything is visible.